iterfzf/__init__.py

```diff
@@ -29,7 +29,7 @@ def iterfzf(
     # Search mode:
     extended=True, exact=False, case_sensitive=None,
     # Interface:
-    multi=False, mouse=True, print_query=False,
+    multi=False, mouse=True, print_query=False, header=None, color=None,
```

Comment:
Could you move `header` parameter to `# Layout:` section?  It looks more appropriate there.